src/mlx90632.c

```diff
@@ -449,4 +471,43 @@ int32_t mlx90632_init(void)
 
     return 0;
 }
+
+int32_t mlx90632_start_measurement_burst(void)
+{
+    int32_t ret;
+    int tries = 150;
+    uint16_t reg;
+
+    ret = mlx90632_i2c_read(MLX90632_REG_CTRL, &reg);
+    if (ret < 0)
+        return ret;
+
+    reg |= MLX90632_CFG_SOB_MASK;
+
+    ret = mlx90632_i2c_write(MLX90632_REG_CTRL, reg);
+    if (ret < 0)
+        return ret;
+
+    while (tries-- > 0)
```

Comment:
Some refactoring of this loop please with `mlx90632_start_measurement`

Reply:
Not entirely sure what you mean? I should make a function to call in both mlx90632_start_measurement() an mlx90632_start_measurement_burst()?